lib/livebook_web/live/hub/edit/personal_component.ex

```diff
@@ -57,6 +63,116 @@ defmodule LivebookWeb.Hub.Edit.PersonalComponent do
             </div>
           </.form>
         </div>
+
+        <div class="flex flex-col space-y-4">
+          <h2 class="text-xl text-gray-800 font-medium pb-2 border-b border-gray-200">
+            Secrets
+          </h2>
+
+          <.secrets_list
+            id="hub-secrets-list"
+            new_secret_path={~p"/hub/#{@hub.id}/secrets/new"}
+            secrets={@secrets}
+            target={@myself}
+          />
+        </div>
+      </div>
+
+      <.modal
+        :if={@live_action in [:new_secret, :edit_secret]}
+        id="secrets-modal"
+        show
+        width={:big}
+        patch={~p"/hub/#{@hub.id}"}
+      >
+        <.live_component
+          module={LivebookWeb.Hub.SecretsComponent}
```

Comment:
Actually let's rename it to something like `SecretEditComponent` or `SecretFormComponent` or similar?